libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tab.component.scss

```diff
@@ -142,6 +173,24 @@
       --sky-override-vertical-tab-box-shadow-focus,
       var(--sky-elevation-focus)
     );
+
+    &.sky-vertical-tab-active {
```

Comment:
This selector fixes a miss on my end when tokenizing vertical tabs. The background color should remain in v2 when the tab is active - it did not.